src/components/layout.css

```diff
@@ -0,0 +1,599 @@
+html {
```

Comment:
This should probably be converted to styled components in your Layout.jsx file, or maybe even the global styles you delcared in your gastby ssr file (depending on if they are actually global styles or styles for elemnts specific to the layout component) ;)